lib/media/track/localvideotrack.js

```diff
@@ -18,7 +18,7 @@ function LocalVideoTrack(mediaStream, mediaStreamTrack, options) {
   if (!(this instanceof LocalVideoTrack)) {
```

Comment:
@ElliotChong `mediaStream` should also be removed from the constructor of `LocalVideoTrack`.